roles/aodh/templates/etc/aodh/policy.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Looks right, I can't see anything here that would be dangerous for cloud_admin to do.